common/defs/solana/programs.json

```
@@ -1359,6 +1359,58 @@
       "name": "Token Program",
       "instruction_id_length": 1,
       "instructions": [
+        {
+          "id": 0,
+          "name": "Initialize Mint",
+          "is_multisig": false,
+          "parameters": [
+            {
+              "name": "decimals",
+              "type": "u8",
+              "optional": false
+            },
+            {
+              "name": "mint_authority",
+              "type": "pubkey",
```

```suggestion
              "type": "authority",
```

Type "authority" means that we display the pubkey only if the account is different from the Trezor account currently being used. I.e. if I'm minting a token, I don't need see that I am the mint authority.

(same for Token2022)